buildkite/run_check_deploy_docker_build.sh

```diff
@@ -4,20 +4,10 @@
 # built successfully. This check is done in the buildkite-assets repo to prevent
 # an unbuildable Dockerfile from being deployed to buildkite instances.
 
-DOCKER_IMAGE_NAME=check-heroku-deploy-image
+echo "--- Building and testing with panorama_command.sh"
+HEROKU_DEPLOY_PATH=deploy/heroku-deploy \
+  HEROKU_DEPLOY_IMAGE_NAME=check-heroku-deploy-image \
+  DEPLOYMENT_APP_NAME=panorama-addons \
+  bash deploy/panorama_command.sh custom
 
-echo "--- Running heroku-deploy build"
-docker build \
-  --build-arg buildkite_agent_uid=$UID \
-  -t $DOCKER_IMAGE_NAME /usr/local/bin/heroku-deploy > build_results.out
-
-EXIT_STATUS=$?
-
-if [ $EXIT_STATUS -ne 0 ]; then
-  echo "+++ Build Results"
-else
-  echo "--- Build Results"
-fi
-cat build_results.out
-
-exit $EXIT_STATUS
+exit $?
```

Comment:
Confirmed that:
* [if the Docker build fails, this step fails](https://buildkite.com/panorama-education/buildkite-assets/builds/304#0184d039-c5b7-4a18-87a8-11ffcb607fdc/122-126)
* [if the Docker build succeeds](https://buildkite.com/panorama-education/buildkite-assets/builds/307#0184d39a-e65c-4d23-9764-6f3244dfe64d/114-183) but [builds an image](https://buildkite.com/panorama-education/buildkite-assets/builds/307#0184d39a-e65c-4d23-9764-6f3244dfe64d/114-149) that [can't run `panorama_command.sh`](https://buildkite.com/panorama-education/buildkite-assets/builds/307#0184d39a-e65c-4d23-9764-6f3244dfe64d/185-186), [this step fails](https://buildkite.com/panorama-education/buildkite-assets/builds/307#0184d39a-e65c-4d23-9764-6f3244dfe64d)
* [If the Docker build succeeds and `panorama_command.sh` succeeds, this step passes](https://buildkite.com/panorama-education/buildkite-assets/builds/306#0184d03f-b73b-478b-acf9-9f3f8fa61e41/126-196)